packages/sync/src/Main.php

```diff
@@ -11,10 +11,16 @@
  * Jetpack Sync main class.
  */
 class Main {
+	static $is_initialized;
+
 	/**
 	 * Initialize the main sync actions.
 	 */
 	public static function init() {
+		if ( self::$is_initialized ) {
```

Comment:
Interesting, I have 3 questions here: 

* Why do we need to do this? Ideally, we would just make sure to initialize it only once.
* This seems like material for another PR, how is it related with this one?
* How do we test the impact of this change?

Reply:
@tyxla When I was working on this, I discovered that init was called on it twice. It made things clearer once it was only invoked once. I'm happy to hold off and submit this in a separate PR if you prefer?

Reply:
> I'm happy to hold off and submit this in a separate PR if you prefer?

That's my preference, but it's not super strong, so I'll defer to you to make the call 😉 I just think it's something that deserves a special attention, dedicated testing, maybe a couple of tests too.